app/src/main/res/values/array.xml

```diff
@@ -74,4 +74,14 @@
         <item>7</item>
         <item>11</item>
     </integer-array>
+    <string-array name="aspect_ratios">
+        <item>16:9 (Switch, Recommended)</item>
+        <item>21:9 (Ultrawide Mods)</item>
+        <item>Device Aspect Ratio (Stretch to fit)</item>
+    </string-array>
+    <integer-array name="aspect_ratios_val">
+        <item>0</item>
+        <item>1</item>
+        <item>2</item>>
+    </integer-array>
```

Comment:
This is redundant as by default IntegerListPreference uses entries' ordinal values when an array for values is not specified.